### PR TITLE
make bem prettier

### PR DESCRIPTION
**Pull request by augustinanders**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -33,20 +33,31 @@ main {
...
-  margin: 10px;
+  list-style: none;
+  display: flex;
+  justify-content: space-around;
+  gap: 5px;
+  margin: 15px 5px 5px;
+  padding: 0;
 }
-.hashtags > a {
+.Card__hashtag {
```
Comment (toshydev) [inline css/styles.css]: I would call this `.Card__hashtag-list-item` or else it looks like it's the parent of `.Card__hashtag-list`.

**Review by toshydev — CHANGES_REQUESTED** [review]
One issue, rest LGTM! ⚡️

File: css/styles.css (hunk 1 above)
Comment (augustinanders) [reply css/styles.css]: Thank you, corrected it now

**Review by Satttoshi — APPROVED** [review]
LGTM :rocket:

**Pull request merged**